content/ops_guides/backup_restore_guide.md

```diff
@@ -136,7 +137,36 @@ This step must take place before the cluster is created and before the snapshot
     1. {'status': 'Snapshot restore in progress...\nThis may take a while, depending on the snapshot size.'}
     1. {'status': 'No `restore_snapshot` workflow currently running.'}
 
+**If the restore is done as part of Upgrade to a newer Cloudify Manager version, consider perform also:**
```

Comment:
If the restore is done as part of Upgrade to a newer Cloudify Manager version, **the following should also be performed:**
(I don't think it's a matter for consideration. workflows which rely on agents won't work correctly)

Reply:
well if no agents installed it will work correctly. That is the reason i wrote consider